### PR TITLE
pricing layout updates.

### LINKED ISSUE #628: updating pricing layout
**What should be improved:**

- the end of the page is very unstructured: unrelated image, contact box, a bit about self-hosting, more links to explore
- if someone is interested in a cloud or self-hosted solution this also effects which information is relevant. Someone who thinks about installing self-hosted probably doesn’t care about a professional open-source plan 
- the pricing plans for cloud and self-hosted look very similar, one even has the same logo. Therefore, the plans can easily be mixed up. 
- we get many questions that would be easy to answer in a FAQ

**Solution:**

- shortening the page to relevant topics
- having different content depending on if one chooses self-hosted or cloud. Other websites with a similar approach https://www.atlassian.com/software/jira/pricing, https://circleci.com/pricing/, https://buddy.works/pricing, https://about.gitlab.com/pricing/
- visualize the differentiation of the plans
- add FAQ

Pricing – 11
Pricing – 15

Could you implement this @nisarhassan12? Feel free to make suggestions if you have other layout ideas. 

Link to the mockup: https://xd.adobe.com/view/624ac883-2198-48b1-42e2-61fc04521425-147b/screen/9dbb0896-e7e7-4f8b-a2f8-25ad1648ce27/Pricing-11

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #628

### CODE REVIEW

File: src/components/pricing/PricingBoxes.tsx [hunk 1]
```diff
@@ -114,99 +116,74 @@ const plans: PricingBoxProps[] = [
...
+                        {selfHostedPlans.map(
+                            (plan, i) => <PricingBox
+                                key={i}
+                                {...plan}
+                                isTitleOutside={true}
+                            />
+                        )}
+                    </div>
                 </div>
+                <p>Do you still have questions? Please, <Link to="/contact/">Get in Touch.</Link></p>
```
Comment (nisarhassan12) [inline src/components/pricing/PricingBoxes.tsx:186]:
```suggestion
                <p>Do you have any questions? Please, <Link to="/contact/">Get in Touch.</Link></p>
```

@ChristinFrohne Can we remove the  IMO it is not required here and does not make much sense. (Sorry If I misunderstood this I don't know English well).

Comment (ChristinFrohne) [reply src/components/pricing/PricingBoxes.tsx:186]: Sure that works too.

**Comment by nisarhassan12** [comment]
>  the backpack icon is wrong- front pocket is missing. Let me know if I can help

@ChristinFrohne Thanks. On the mockup, both the pocket and the backpack are different icons i.e I can Download both separately but not together as one. Please provide me with the full icon as one. Thanks.

**Comment by nisarhassan12** [comment]
@ChristinFrohne I have made the requested changes please have a look whenever you'll have time. Thanks.

**Comment by ChristinFrohne** [comment]
Thanks @nisarhassan12 

Some small nits:

- [ ] Any reason why you don't display the cloud in self-hosted? 

- [ ] can you adjust the box to its content -> shorten it a bit

Bildschirmfoto 2020-05-13 um 07 59 25


- [ ] Can you make the font one style smaller? Currently, it's bigger than the headlines and a bit distracting
Bildschirmfoto 2020-05-13 um 08 00 00

- [ ] Could you format this text? Some characters also got mixed up `efforts.', 'If you`
Bildschirmfoto 2020-05-13 um 08 03 38

- [ ] `docs`button in self-hosted should link to https://www.gitpod.io/docs/self-hosted/latest/self-hosted/

- [ ] is it possible to get back to Pricing/self-hosted if clicking on the back button? Currently, it always takes me back to pricing/cloud

- [ ] could you move the icons a bit further down?  Currently, they are closer to the border than to their belonging headline
Bildschirmfoto 2020-05-13 um 08 08 45

- [ ] have a bit less spacing between the headline  `special use cases` and the two boxes
Bildschirmfoto 2020-05-13 um 08 13 50

- [ ] could you add more spacing between the last element and the footer on mobile view 
Bildschirmfoto 2020-05-13 um 08 16 36


- [ ] I just noticed that `Self-Hosted FAQ` makes more sense than ´Self-Hosted Pricing FAQ´ as the questions don't all relate to pricing

**Comment by nisarhassan12** [comment]
>  - [x] Any reason why you don't display the cloud in self-hosted?

Sorry! I have added it now and it works fine before in https://github.com/gitpod-io/website/pull/613#discussion_r420495171 I was not able to get it right.

**Comment by nisarhassan12** [comment]
> - [x] can you adjust the box to its content -> shorten it a bit

@ChristinFrohne I have reduced the height for the self-hosted pricing plans Is that what you meant or you asked for reducing the width ?

**Comment by nisarhassan12** [comment]
@ChristinFrohne I have made the changes please have a look whenever you'll have time.

**Comment by ChristinFrohne** [comment]
Great, thanks. Just two more nits.

- [ ] could you move both buttons a bit further down
Bildschirmfoto 2020-05-14 um 15 39 09

- [ ] could you add more spacing between "create team" and the question mark?
Bildschirmfoto 2020-05-14 um 15 42 13

**Comment by ChristinFrohne** [comment]
@meysholdt could you have a look at the FAQ for self-hosted?

**Pull request merged**